esp/test/httptest/main.cpp

```diff
@@ -17,6 +17,9 @@
 
 #include "httptest.hpp"
 
+constexpr unsigned TIMES_LIMIT = 10000000; //10M
```

Comment:
@AttilaVamos Can you comment.  I'm not sure where httptest is used to know what these limits should be used.  It's just a parameter check.  If httptest is meant to be able to do long stress testing I could see high values being desirable.  Just not sure what values.

Reply:
@afishbeck The Coverity scan is a static analysis. If the httptest is a stand-alone test and can run long-term (how and how long?) then I can add it to OBT.

@wangkx The Coverity scan executed once a week (limitation of free scan) on Monday with master branch.